release_notes.py

```diff
@@ -165,13 +169,23 @@ def load_data(self, data):
     def get_header(self):
         return "Scripts"
 
-    def added_release_notes(self, cnt):
+    def added_release_notes(self, cnt, demisto_version, first_release):
+        if first_release:
+            from_version = cnt.get("fromversion") or cnt.get("fromVersion")
+            if from_version:
+                if LooseVersion(from_version) == LooseVersion(demisto_version):
```

Comment:
if `from_version < demisto_version` we also want to include release notes (true all `added_release_notes`)

we have a lot of code duplication here
I recommend create this logic before calling `added_release_notes` / `modified_release_notes`